dash/frontend/src/app/modules/private/pages/cluster/cluster-info/cluster-info.component.ts

```diff
@@ -68,24 +86,15 @@ export class ClusterInfoComponent implements OnInit {
     });
   }
 
-  @HostListener('window:resize', ['$event'])
-  calculateScreenSize($event?: any) {
-    this.scrWidth = window.innerWidth;
-    this.screenSizeCollapse(this.scrWidth);
-  }
-
-  set scrWidth(val: number) {
-    if (val !== this.width) {
-      this.width = val;
-    }
-  }
-
-  get scrWidth(): number {
-    return this.width;
+  ngAfterViewInit() {
+    this.isHandsetOrXS$.subscribe((newIsHandsetOrXS) => {
+      this.isMobileDevice = newIsHandsetOrXS;
```

Comment:
you probably want a .pipe(takeUntil(this.unsubscribe$) here.

Also, since it looks like this is the only place you are using the isHandsetOrXS$, you probably don't need the shareReplay() (or to have it as a class property)

Reply:
@kyle-intelletive - updated. Could you take another look?